File: problema7.py
# Problema 7
puntuacion = int(input("Ingresa la puntuacion del empleado: "))
if puntuacion >= 10:
    print("Excelente, recibe $200000")
elif puntuacion >= 7:
    print("Bueno, recibe $100000")
elif puntuacion >= 5:
    print("Meh, recibe $50000")
else:
    print("Pongase a trabajar, no recibe recompensa")

File: problema7.hs
-- Problema 7
evaluarEmpleado :: Int -> String
evaluarEmpleado p
    | p >= 10 = "Excelente, recibe $200000"
    | p >= 7 = "Bueno, recibe $100000"
    | p >= 5 = "Aceptable, recibe $50000"
    | otherwise = "Pongase a trabajar, no recibe recompensa"